sibylapp2/view/plots/charts.py

```diff
@@ -2,7 +2,16 @@
 import plotly.express as px
 
 
-def plot_temporal_line_charts(df: pd.DataFrame, y_max=1.0, y_min=-1.0, x_max=30, x_min=0):
+def plot_temporal_line_charts(
```

Comment:
The plots don't align on the x-axis, with the legend added, which makes it harder to follow the changes. I think this can be fixed by either setting a fixed size for both plots, or thinking of ways to plot on a single graph (maybe using two sets of y-axes)
The second may be preferable actually, so you don't have to look up and down between two plots

Reply:
The new update makes the plot overlap with each other. Let me know what you think about the new look!

Reply:
I like the overlap more - its easier to link the contributions to the output. 

Main feedback at this point is:
1. The color representing "failure" and "success" changes between turbines (if a turbine only has success, the dots will be the same color as when a turbine has only failure). The best way to fix this is to just show both on the legend always. This PR may be helpful to see how we've fixed this before: https://github.com/sibyl-dev/sibylapp2/pull/40 
2. Right now success and failure are shown in very similar shades of blue, we should make them distinct. To avoid being blocked we can focus on categorical predictions for now